.github/workflows/build.yml

```
@@ -7,6 +7,7 @@ jobs:
       fail-fast: false
       matrix:
         rails-version: [6, 7]
+        ruby-version: [2.7.6, 3.0.4, 3.1.2]
```

```suggestion
        ruby-version: [2.7, 3.0, 3.1]
```

While this gives us fine-grained control over the exact Ruby versions used in a given test run, I've found in the past that it's extremely tedious to be constantly updating the workflow file whenever new Ruby versions come out, to very little benefit.  The `ruby/setup-ruby` action accepts the MAJ.MIN form quite happily, according to the docs.